grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcExceptionHandler.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.grpc;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.linecorp.armeria.common.grpc.GrpcStatusFunction;
+
+/**
+ * Specifies an {@link GrpcStatusFunction} class which handles exceptions throwing from an
+ * annotated gRPC service or its methods.
+ */
+@Repeatable(GrpcExceptionHandlers.class)
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE, ElementType.METHOD })
+public @interface GrpcExceptionHandler {
+
+    /**
+     * {@link GrpcStatusFunction} implementation type. The specified class must have an accessible
+     * default constructor.
+     */
+    Class<? extends GrpcStatusFunction> value();
```

Comment:
What do you think of deprecating `GrpcStatusFunction` and adding `GrpcExceptionHandlerFunction` to align with `ExceptionHandler` and `ExceptionHandlerFunction`?
Also, the `GrpcStatusFunction` name itself is unclear because it doesn't seem to have anything to do with an exception. 😉

Reply:
I agree with you.
I addressed this suggestion